Add --force argument to php artisan backpack:crud

karandatwani92 opened the pull request:
## WHY
To address this #186 

### AFTER - What is happening after this PR?
Added `--force` argument to `php artisan backpack:crud`

### Is it a breaking change or non-breaking change?
Non-breaking

### How can we test the before & after?
`php artisan backpack:crud item`
`php artisan backpack:crud item --force`

tabacitu commented:
Awesome! Is this ready @karandatwani92 ? If so, please assign @pxpm to review, test and merge.

karandatwani92 commented:
Yes @tabacitu , this is ready for review.

Hey @pxpm,  Please give it a read and test? Thanks 🙃

pxpm commented on src/Console/Commands/CrudBackpackCommand.php:
```diff
@@ -52,14 +52,14 @@ public function handle()
 
         // Create the CRUD Model and show output
-        $this->call('backpack:crud-model', ['name' => $name]);
+        $this->call('backpack:crud-model', ['name' => $nameTitle, '--force' => $this->option('force')]);
```
We can probably store the `--force` option up close where we init other command properties like `$name`, `$nameTitle` etc. We then re-use the variable across the command 👍 This is just personal preference, it's not that is bad implemented.

```suggestion
        $this->call('backpack:crud-model', ['name' => $nameTitle, '--force' => $force]);
```

pxpm replied:
Please also take a look at this @karandatwani92 let me know if you think we shouldn't do this, otherwise please do it 👍

karandatwani92 commented:
@pxpm,

I didn't knew backpack has following signatures:
`php artisan backpack:model`
`php artisan backpack:request`

And this doesn't exists
`php artisan backpack:controller`


[Docs](https://backpackforlaravel.com/docs/5.x/generating-code#generate-crud-files-1) only shows the following which i have already covered:

Command | Description.
-- | --
php artisan backpack:crud-controller | Generate a Backpack CRUD controller.
php artisan backpack:crud-model | Generate a Backpack CRUD model
php artisan backpack:crud-request | Generate a Backpack CRUD request
php artisan backpack:crud-operation | Generate a custom Backpack CRUD operation trait

karandatwani92 commented:
I found these two were unmaintained and replaced by newer commands.
So, as a final solution i have refactored these two to call newer backpack commands to keep up with newer functionalities.
```
php artisan backpack:model
php artisan backpack:request
```

karandatwani92 commented:
Hi @pxpm
I rechecked, both used to do the same thing with no differences. So I think you can merge.
